src/components/TextMaskedInput/TextMaskedInput.js

```diff
@@ -9,6 +9,7 @@ import cleanse from '../../helpers/cleanse.js'
 
 import styles from '../TextInput/TextInput.module.scss'
 import errorStyles from '../Errors.module.scss'
+import { FontAwesomeIcon } from '@fortawesome/react-fontawesome'
 
 export const TextMaskedInput = (props) => {
```

Comment:
@MayUWish could we please include some examples using markdowns similar to what you did for `NumberInput` and `EmailInput` ?